src/MainWindow/main_window.cpp

```diff
@@ -147,6 +147,13 @@ void MainWindow::SetWindowTitle(const QString& filename, const QString& project,
   }
 }
 
+void MainWindow::CloseOpenedTabs() {
+  while (TextEditorForm::Instance()->GetTabWidget()->count() != 0) {
```

Comment:
for better readability it is better to do something like this:
`auto tabWidget = TextEditorForm::Instance()->GetTabWidget();`
`while (tabWidget->count() != 0) {...}`

Reply:
Fixed.